.github/workflows/cifuzz.yml

```diff
@@ -0,0 +1,26 @@
+name: CIFuzz
+on: [pull_request]
+jobs:
+  Fuzzing:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Build Fuzzers
+      id: build
+      uses: google/oss-fuzz/infra/cifuzz/actions/build_fuzzers@master
+      with:
+        oss-fuzz-project-name: 'toml_edit'
+        dry-run: false
+        language: rust
+    - name: Run Fuzzers
+      uses: google/oss-fuzz/infra/cifuzz/actions/run_fuzzers@master
+      with:
+        oss-fuzz-project-name: 'toml_edit'
+        fuzz-seconds: 300
```

Comment:
With having a cap on the maximum number of parallel runners, I feel like 300 seconds is too much and would negatively affect CI times

Reply:
Apologies for the delay @epage -- I'm happy to address this: would would your preference be? We could go down to 15 seconds?

Reply:
15s shouldn't be a problem

Reply:
Sounds good! Fixed it and updated to the lastest CIFuzz action as well